src/client/components/footer/Footer.module.css

```diff
@@ -17,5 +15,15 @@
   width: 1200px;
   background-color: rgba(66,66,66, 0.2);
   text-align: center;
+  color: red;
+  font-family: Montserrat;
+  font-size: 20px;
+  font-weight: 500;
+  letter-spacing: 0;
+  line-height: 24px;
+}
+
+.footer span{
```

Comment:
use instead of this className, don't use tag name for styling

Reply:
Done)